spec/support/concerns/organisations.rb

```diff
@@ -0,0 +1,55 @@
+RSpec.shared_examples "it can have organisations" do |schema|
+  before do
+    @example_doc = GovukSchemas::Example.find(schema, example_name: schema)
```

Comment:
I think `@example_doc` should be renamed to `@content_store_response` for consistency with the other RSpec files.

Reply:
FYI, there's a BIG PR about changing the existing spec files - @georges1996 it would probably be useful if you temporarily added the rspec rubocop config and just run it over the files you're touching/adding in this PR.

```# .rubocop.yml
inherit_gem:
  rubocop-govuk:
    ...
    - config/rspec.yml